### PR TITLE
git-ignore: add page (and move former to git-ignore-io)

### PR DESCRIPTION
**Pull request by nicokosi**
1. Move existing `git-ignore` pages to `git-ignore-io` ;
2. Create a "true" `git-ignore` page.

See the two distinct commits. 👓 

- [x] The page (if new), does not already exist in the repo. (kind of 😇)
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Related: #5137

### CODE REVIEW

File: pages/common/git-ignore-io.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# git ignore-io
```
Comment (nicokosi) [inline pages/common/git-ignore-io.md:1]: Is there any issue with the "command-subcommand-subcommand" pattern? In fact, `git-ignore` is a command on its own (it does not refer to the `git ignore io` command).

**Comment by nicokosi** [comment]
> The [build](https://github.com/tldr-pages/tldr/actions/runs/819231447) for this PR failed with the following error(s):
> 
> ```
> pages/common/git-ignore.md:13: TLDR011 Page never contains more than a single empty line
> ```
> 
> Please fix the error(s) and push again.

Hello 🤖 , it's fixed! ✅ 
PS: I have to fix my pre-commit hook that does not work anymore... 🤔

File: pages/common/git-ignore-io.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/git-ignore-io.md:1]: Yeah, we have issues with this.  The following `tldr` invocations would all display the page `git-ignore-io.md`:
```
tldr git ignore io
tldr git-ignore-io
tldr git-ignore io
tldr git ignore-io
```
The title of the page should be however the command would be invoked on the actual command-line.

File: pages/common/git-ignore.md [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-ignore>.
 
-- List available templates:
+- Show global and local `.gitignore` files' content:
 
-`git ignore list`
+`git ignore`
 
-- Generate a .gitignore template:
+- Ignore file(s) privately, updating `.git/info/exclude` file:
```
Comment (bl-ue) [inline pages/common/git-ignore.md:10]: Interesting, so you can ignore files without updating `.gitignore` 🤔

**Review by bl-ue — APPROVED** [review]

File: pages/common/git-ignore.md [hunk 3]
```diff
@@ -1,12 +1,20 @@
 # git ignore
 
-> Generate .gitignore files from predefined templates.
-> More information: <https://docs.gitignore.io/install/command-line>.
+> Show/update `.gitignore` files.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-ignore>.
 
-- List available templates:
+- Show global and local `.gitignore` files' content:
```
Comment (marchersimon) [inline pages/common/git-ignore.md:6]:
```suggestion
- Show the content of  all global and local `.gitignore` files:
```
I think this is a bit more readable. Also, does `global` mean, that it changes it for all git repos?

Comment (CleanMachine1) [reply pages/common/git-ignore.md:6]: Not to sure if this is correct however, is `global` indicating all ALL branches have it ignored

**Review by marchersimon — APPROVED** [review]

File: pages/common/git-ignore.md [hunk 4]
```diff
@@ -1,12 +1,20 @@
 # git ignore
 
-> Generate .gitignore files from predefined templates.
-> More information: <https://docs.gitignore.io/install/command-line>.
+> Show/update `.gitignore` files.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-ignore>.
```
Comment (sbrl) [inline pages/common/git-ignore.md:4]: This doesn't look like a core git command. I recommend a 2nd description line that includes this information - e.g. `Part of SOMETHING_HERE`:
```suggestion
> Show/update `.gitignore` files.
> Part of SOMMETHING_HERE. See also `git ignore-io`.
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-ignore>.
```

**Review by sbrl — CHANGES_REQUESTED** [review]
`git ignore

File: pages/common/git-ignore.md (hunk 4 above)
Comment (bl-ue) [reply pages/common/git-ignore.md:4]: `git-extras`.  I'll update your suggestion.

**Review by sbrl — APPROVED** [review]
Thanks, @nicokosi!
